rudder-inventory-ldap/debian/rules

```diff
@@ -9,23 +9,21 @@
 # Uncomment this to turn on verbose mode.
 export DH_VERBOSE=1
 
-
-
-
 configure: configure-stamp
 configure-stamp:
 	dh_testdir
 	# Add here commands to configure the package.
-	cd SOURCES/openldap-source && ./configure --prefix=/opt/rudder --enable-dynamic --enable-debug --enable-modules --enable-hdb=mod --enable-monitor=mod --enable-dynlist=mod --with-cyrus-sasl
+	cd SOURCES/berkeleydb-source/build_unix && ../dist/configure --prefix=/opt/rudder
+	export LD_LIBRARY_PATH="/opt/rudder/lib" ; CPPFLAGS="-I/opt/rudder/include" ; export LDFLAGS="-L/opt/rudder/lib" ; export LIBS="-L/opt/rudder/lib" ; cd SOURCES/openldap-source && ./configure --prefix=/opt/rudder --libdir=/opt/rudder/lib/ldap --enable-dynamic --enable-debug --enable-modules --enable-hdb=mod --enable-monitor=mod --enable-dynlist=mod --with-cyrus-sasl
```

Comment:
A more posix compliant version would be :

cd SOURCES/openldap-source && VAR1=bla VAR2=bla ./configure ...